geth/utils/filesystem.py

```diff
@@ -1,8 +1,7 @@
+import errno
 import os
-import sys
 import shutil
-import errno
-
+import sys
 
 if sys.version_info.major == 2:
     FileNotFoundError = OSError
```

Comment:
Also can go. Maybe we should make an issue to remove all these since this PR is so big already?